packages/app/styles/global.css

```diff
@@ -198,6 +198,35 @@ body {
     }
 }
 
+/* used to add a fancy highlight animation in TaskMarkingSelector */
+.highlight-spin:before {
+    content: '';
+    position: absolute;
+    left: -2.5px;
```

Comment:
I feel like we stepped on the same issues with before with using `px` should we account for mobile version?

Reply:
in this instance where it's just adding a small visual effect that extends just beyond the elements size, it's actually a better idea to use absolute units like `px` since otherwise it will increase in size with relative units. 

Also, I tested with it on mobile and also with it on 400% zoom, it looks fine.